deploy/cloud/data/package_policy_cspm_gcp.json

```diff
@@ -0,0 +1,70 @@
+{
+  "policy_id": "${agent_policy_id}",
+  "package": {
+    "name": "cloud_security_posture",
+    "version": "1.5.2-preview2"
+  },
+  "name": "pkg-plcy-cspm-gcp-tf",
+  "description": "",
+  "namespace": "default",
+  "inputs": {
+    "kspm-cloudbeat/cis_k8s": {
+      "enabled": false,
+      "streams": {
+        "cloud_security_posture.findings": {
+          "enabled": false
+        }
+      }
+    },
+    "kspm-cloudbeat/cis_eks": {
+      "enabled": false,
+      "streams": {
+        "cloud_security_posture.findings": {
+          "enabled": false,
+          "vars": {}
+        }
+      }
+    },
+    "cspm-cloudbeat/cis_aws": {
+      "enabled": false,
+      "streams": {
+        "cloud_security_posture.findings": {
+          "enabled": false,
+          "vars": {
+            "aws.credentials.type": "cloud_formation",
+            "aws.account_type": "organization-account"
+          }
+        }
+      }
+    },
+    "cspm-cloudbeat/cis_gcp": {
+      "enabled": true,
+      "streams": {
+        "cloud_security_posture.findings": {
+          "enabled": true,
+          "vars": {}
```

Comment:
gcp will have these vars:

https://github.com/elastic/integrations/pull/7403/files#diff-1977b9299340bb9ac9f1d5c15a837593d3bdce0053da787b2df1a906788e11e4R126-L148

Reply:
I'm not entirely sure what you mean. For the current task of deploying CSPM GCP using Shell Cloud, we don't require these variables. After configuring all the necessary details, I generated a pre-request, and it does not include these variables.